cfme/tests/intelligence/test_reports_with_timelines.py

```diff
@@ -43,9 +148,15 @@ def test_custom_reports_with_timelines(report):
         testSteps:
             1. Add an infrastructure provider to ensure that an event for each report will occur
             2. Navigate to Cloud Intel -> Reports
-            3. Create a custom report for each of the parameters
+            3. Modify one of the default reports to display events from today
             4. Navigate to Cloud Intel -> Timelines
             5. Click on the report you created
             6. Verify that the timeline (with events) is properly displayed
     """
-    pass
+    copied_report = setup_for_reports(
```

Comment:
Q: what is the reason for making a bunch of such functions instead of fixtures ?

Reply:
@izapolsk this mainly has do with "delayed execution", I only want these functions to run during the test run, similar to `setup_for_alerts` in `cfme/control/test_alerts.py`

Reply:
Every automated test usually has 3 phases: 1. setup; 2. test run; 3. teardown;
We use py.test as test runner. It provides and firmly recommends using fixtures approach for both setup and teardown.

So, I'd prefer you moved this setup/teardown code to appropriate fixtures.
I could help you to break this code down into several fixtures if you wish.